### ISSUE TITLE
NTR: Modifier: Hemorrhagic nature to skin lesions (Skin and mucosa)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8003]
New term request Modifier: Hemorrhagic nature to skin lesions (elthat_100807144826):   UMLS: 3670842

**Comment by pnrobinson** [comment]
@MickeySegal
what is the context?
How is this different from 
[Subcutaneous hemorrhage](https://hpo.jax.org/app/browse/term/HP:0001933)
?

**Comment by MickeySegal** [comment]
It is not bleeding under the skin.  It is skin lesions that tend to bleed.

**Comment by pnrobinson** [comment]
This is a little hard to model; could you give a concrete example?

**Comment by pnrobinson** [comment]
@MickeySegal 
So you mean a lesion that is bleeding externally rather than into the skin (Dermal hemorrhage)?

**Comment by MickeySegal** [comment]
Dermal hemorrhage just describes bleeding without associating it with a skin lesion that bleeds, such as those described above.  In frostbite the bleeding is into blisters, that become blood-filled.  In epidermolysis bullosa at the URL above the description is "Blisters may be large, hemorrhagic, and severe".  It sounds like the bleeding is into the lesions, not the outside, and not in areas unassociated with lesions.

**Comment by pnrobinson** [comment]
@MickeySegal 
sorry for the delay

HP:0020011
label: Bleeding into skin lesions
def: Presence of bleeding within (pre-existing) cutaneous lesions such as blisters or vesicles, resulting in blood-filled lesions.
